sdk.js

```diff
@@ -301,6 +303,28 @@ class PaddleSDK {
 			subscription_id: subscriptionID,
 		});
 	}
+
+	/**
+	 * Generate a custom pay link 
+	 *
+	 * @method
+	 * @param {object} body
+	 * @returns {Promise}
+	 * @fulfil {object} - The new pay link url 
+	 *
+	 * @example
+	 * const custom = await client.generatePayLink({
+	 *  "title" : "my custom checkout",
+	 *  "custom_message" : "some custom message"
+	 * 	"prices": [
+	 *		"USD:19.99",
+	 *		"EUR:15.99"
+	 *	 ]
+     *	});
+	 */
+	generatePayLink(body) {
+		return this._request('/product/generate_pay_link', body , form = false);
```

Comment:
This is invalid. I think you want this:

```js
return this._request('/product/generate_pay_link', body, {}, false);
```

Reply:
I guess you are right, didn't have time to test. If anyone is willing to test that would be great